jenkins/ompi/ompi_test.sh

```diff
@@ -110,265 +103,226 @@ mlnx_cov="SKIP"
-fi
-
-if [ -n "$NOJENKINS" -a -d $OMPI_HOME1 ]; then
-    jenkins_test_build=no
-    jenkins_test_cov=no
-    jenkins_test_check=no
-    jenkins_test_src_rpm=no
-    jenkins_build_passed=1
-fi
-
-# check for jenkins commands in PR title
-if [ -n "$ghprbPullTitle" ]; then
-    check_commands "$ghprbPullTitle"
+rel_path=$(dirname "$0")
+abs_path=$(readlink -f "${rel_path}")
+
+if [ "${ci_test_debug}" = "no" ]
+then
+    ci_test_threads="yes"
+    ci_test_oshmem="yes"
+    ci_test_src_rpm="no"
+    mlnx_cov="SKIP"
+else
+    ci_test_build="yes"
+    ci_test_examples="yes"
+    ci_test_oshmem="yes"
+    ci_test_ucx="yes"
+    ci_test_vg="yes"
+    ci_test_check="no"
+    ci_test_src_rpm="no"
+    ci_test_help_txt="no"
+    ci_test_threads=""
+    ci_test_cov="no"
+    ci_test_known_issues="no"
+    ci_test_slurm="no"
+    ci_test_comments="no"
 fi
 
-# check for jenkins command in PR last comment
-if [ -n "$ghprbPullLink" ]; then
-    set +eE
-    pr_url=$(echo $ghprbPullLink | sed -e s,github.com,api.github.com/repos,g -e s,pull,issues,g)
-    pr_url="${pr_url}/comments"
-    pr_file="$WORKSPACE/github_pr_${ghprbPullId}.json"
-    curl -s $pr_url > $pr_file
-    echo Fetching PR comments from URL: $pr_url
-
-    # extracting last comment
-    pr_comments="$(cat $pr_file | jq -M -a '.[length-1] | .body')"
-
-    echo Last comment: $pr_comments
-    if [ -n "$pr_comments" ]; then
-        check_commands "$pr_comments"
-    fi
-    set -eE
+if [ -d "${OMPI_HOME1}" ]
+then
+    echo "WARNING: ${OMPI_HOME1} already exists"
+    ci_test_build="no"
+    ci_test_cov="no"
+    ci_test_check="no"
+    ci_test_src_rpm="no"
+    ci_build_passed="1"
 fi
 
+echo "Running following tests:"
+set | grep ci_test_
 
-echo Running following tests:
-set|grep jenkins_test_
+extra_conf=${extra_conf:=""}
 
-if [ "$jenkins_test_threads" = "yes" ]; then
-    extra_conf="--enable-mpi-thread-multiple --enable-opal-multi-threads $extra_conf"
+if [ "${ci_test_threads}" = "yes" ]
+then
+    extra_conf="--enable-mpi-thread-multiple --enable-opal-multi-threads ${extra_conf}"
 fi
 
-if [ "$jenkins_test_slurm" = "yes" ]; then
-    extra_conf="--with-slurm --with-pmi $extra_conf"
+if [ "${ci_test_slurm}" = "yes" ]
+then
+    extra_conf="--with-slurm --with-pmi ${extra_conf}"
 fi
 
 function mpi_runner()
 {
     AFFINITY=${AFFINITY_GLOB}
-    if [ "$1" = "--no-bind" ]; then
+
+    if [ "$1" = "--no-bind" ]
+    then
         AFFINITY=""
         shift
     fi
 
-    local np=$1
+    local np="$1"
     local exe_path="$2"
-    local exe_args=${3}
-    local common_mca="--bind-to none --mca orte_tmpdir_base $jenkins_session_base"
-    local mpirun="$OMPI_HOME/bin/mpirun"
-
-    local has_timeout=$($OMPI_HOME/bin/mpirun --help | grep timeout | wc -l)
-    if [ "$has_timeout" = "0" ]; then
+    local exe_args="$3"
+    local common_mca="--bind-to none --mca orte_tmpdir_base ${ci_session_base}"
+    local mpirun="${OMPI_HOME}/bin/mpirun"
+    local has_timeout
```

Comment:
lines 192-203 can be removed . it was needed during time when mpirun builtin timeout mechanism was present in some branches only, now it presents in all.

Reply:
Fixed.